cico_build.sh

```diff
@@ -69,17 +71,31 @@ function deploy() {
   if [ -n "${QUAY_USERNAME}" ] && [ -n "${QUAY_PASSWORD}" ]; then
     docker login -u "${QUAY_USERNAME}" -p "${QUAY_PASSWORD}" "${REGISTRY}"
   else
-    echo "Could not login, missing credentials for the registry"
+    echo "Could not login, missing credentials for pushing to the 'openshiftio' organization"
   fi
 
-  # Let's deploy
+  # Let's push 'rhel' images to 'quay.io/openshiftio'
   docker build -t ${IMAGE} -f ${DOCKERFILE} .
 
   TAG=$(echo "$GIT_COMMIT" | cut -c1-"${DEVSHIFT_TAG_LEN}")
 
   tag_push "${REGISTRY}/openshiftio/$IMAGE:$TAG"
   tag_push "${REGISTRY}/openshiftio/$IMAGE:latest"
-  echo 'CICO: Image pushed, ready to update deployed app'
+  echo "CICO: Images pushed to 'quay.io/openshiftio', ready to update deployed app"
```

Comment:
A few suggestions:
- I don't think you should login/push to `quay.io/eclipse-che` if the `"$TARGET" == "rhel" `
- You don't need to the `docker build` twice
- I would avoid tagging the images as `latest` because that should be done when we do the release of Che every 3 weeks (especially for eclipse-che org)

Reply:
And do we really need to push centos images to both openshiftio and eclipse-che?

Reply:
> I don't think you should login/push to quay.io/eclipse-che if the "$TARGET" == "rhel"
> You don't need to the docker build twice

Maybe I'm missing smth but in the issue description there is - `Update che-devfile-registry-master CentOS CI job to push rhel images to quay.io/openshiftio and centos images to quay.io/eclipse-che` and my understanding is that master CI job always build 'rhel for now, right?. So, I do not really understand how we can avoid building and pushing images twice if we want CentOS images beeing build as part of the same job

Reply:
it is actually related to my question in my first question in the PR - https://github.com/eclipse/che-devfile-registry/pull/31#issuecomment-510394593

Reply:
If what you are saying was true ($TARGET is always "rhel") the script would build `rhel` images **only** (the script does only one docker build). But we know that this is not the case because we see that centos images are pushed to the registry right? 

So I think that this script is called twice: once with not specific $TARGET and once with TARGET=rhel. And we should double check with SD team but I suspect that we (you and I) do not have the privileges to see the rhel images even if they are pushed in that org.

Reply:
Another thing I would add in this PR is an update to the README file to use the `eclipse-che` org images instead of the openshiftio.

Reply:
> So I think that this script is called twice: once with not specific $TARGET and once with TARGET=rhel

@l0rd looks like this is the case (script is called twice with & without the target), because:

- for now there is only one job for devfile registry - https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L4470
- centos images are pushed to https://quay.io/repository/openshiftio/che-devfile-registry?tab=tags
- on dsaas devfile-registry is build against `openshiftio/rhel-che-devfile-registry` images (and yes, we do not have access to https://quay.io/repository/openshiftio/rhel-che-devfile-registry)

Reply:
Yep.. The script is run twice. Once here:
https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L891
And for the second time here (where `extra_target=rhel`):
https://github.com/openshiftio/openshiftio-cico-jobs/blob/master/devtools-ci-index.yaml#L902